iotfunctions/pipeline.py

```diff
@@ -2286,13 +2286,15 @@ def execute(self, df=None, to_csv=False, dropna=False, start_ts=None, end_ts=Non
         # added as the ui expects each stage to contribute one or more output items
         for pl in preloaded_item_names:
             df[pl] = True
-        for s in stages:
+        for counter, s in enumerate(stages):
```

Comment:
This change only creates a new variable for logging.
I'm still not clear where the chunking is occurring.

Reply:
The vast majority of changes is in analytics-service-pipeline. See Pull request https://github.ibm.com/IoT-Enterprise/analytics-service-pipeline/pull/269